doc/40_Configuration.md

```diff
@@ -13,7 +13,7 @@ Gitpod workspaces come with good defaults, but of course not every workspace loo
 
 ## `.gitpod` File
 
-A workspace gets configured through a `.gitpod` file written in `YAML` syntax. There are three ways
+A workspace gets configured through a `.gitpod` file written in the YAML syntax. There are three ways
```

Comment:
The 'the' here isn't really neccesary I think ... something is written in _German syntax_, not the _German syntax_ (a bit of a forced example, I realise)

Reply:
right. Googling it in *.co.uk just confirms your opinion :-)